mesh/accounts/views.py

```diff
@@ -1,3 +1,16 @@
-from django.shortcuts import render
+from django.views.decorators.csrf import csrf_exempt
+from django.http import JsonResponse
+from rest_framework.views import exception_handler
 
-# Create your views here.
+@csrf_exempt
+def bio_view(request):
+    if request.method == "POST":
+        data = request.POST
+        if 'biography' not in data:
+              return JsonResponse({'error': 'Invalid request. Missing biography field.'}, status=400)
+        else:
+            print(data.get('biography'))
+            # Save Biography to Database
+            return JsonResponse({'message': 'biography saved successfully'}, status=200)
+    else:
+        return JsonResponse({'error': 'Method not allowed'}, status=405)
```

Comment:
better error message may be to specify which method they used rather than just "method not allowed". e.x. get method not allowed.